src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -93,6 +93,13 @@ public void removePerson(Person key) {
         persons.remove(key);
     }
 
+    /**
+     * Clears every entry from this {@code AddressBook}.
+     */
+    public void clearAllPerson() {
+        persons.clearAllPerson();
+    }
+
```

Comment:
Not sure if we should change this @code AddressBook part, we might be able to refactor this all at once in the future